resources/schemas/org.gnome.shell.extensions.rounded-window-corners-reborn.gschema.xml

```diff
@@ -35,8 +35,13 @@
       <default>0</default>
     </key>
 
-    <key name="border-color" type="(dddd)">
```

Comment:
If you're changing prefs schema, you should update the schema version in `utils/settings.ts`, and if you remove some keys you have to reset them in that same file (it has a function for that)